documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -0,0 +1,46 @@
+---
+title: State Management
+---
+
+Managing state is one of the hardest parts in application development. This section covers various use cases with regards to state management and what to watch out for.
+
+## Avoid global state in SSR
+
+If you are creating an [SPA](glossary#csr) with SvelteKit, you can create global state freely, as you can be sure that it's only initialized inside the user's browser. If you use [SSR](glossary#ssr) however, you have to watch out for a couple of things when managing state. In many server environments, a single instance of your app will serve multiple users (this is not specific to SvelteKit - it's one of the gotchas of working with such environments). For that reason, per-request or per-user state must not be stored in global variables.
```

Comment:
I read SPA as the word "spa", so "an spa" sounds weird to me - though it'd be correct for folks reading it as s-p-a. Maybe we just write out "single page application" to avoid the issue
```suggestion
If you are creating a [single page application (SPA)](glossary#csr) with SvelteKit, you can create global state freely, as you can be sure that it's only initialized inside the user's browser. If you use [SSR](glossary#ssr) however, you have to watch out for a couple of things when managing state. In many server environments, a single instance of your app will serve multiple users (this is not specific to SvelteKit - it's one of the gotchas of working with such environments). For that reason, per-request or per-user state must not be stored in global variables.
```

Reply:
I linked there so people can look up good to enter spa mode

Reply:
doesn't `glossary#spa` better describe how to enter SPA mode than the CSR entry does? Neither says directly, but the SPA entry directs users to `adapter-static#spa-mode`